sample/golang-echo/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.9-alpine
```

Comment:
of course, all the older stable container tags are unsupported AND vulnerable. For now, I'm setting this to `golang:1.9-alpine3.6`
would that be okay. If not, got a preferred golang container for this sample? Or I could just scrap it for fsharp instead